UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -193,13 +193,37 @@ <h4>Metadata techniques</h4>
 			
 			<section id="visual-adjustments">
 				<h3>Visual Adjustments</h3>
+				<p>A Yes/No/Unknown answer to the question "Can a user change the aspect of textual contents". </p>
+
+				<p>Indicates if the appearance of the text and the page layout can be modified according to the possibilities offered by the reading system.</p>
 				
-				<p>TBD</p>
-				
-				<p>TBD</p>
+				<p>Readers with visual impairment or cognitive disabilities rely on the ability to change color of background and texts,  font family and font size as well as spaces between letters, words, sentences or paragraphs.</p>
+				<p><!--this one might be moved to the technics-->The reflowable information alone is not sufficient to make sure that modifications operated will not result on text overlaying or unreadable colors.</p>
+			
 				<h4>Examples</h4>
-                <p>TBD</p>
-                <h4>Metadata techniques</h4>
+                <p>The examples are provided as long and short sentence for flexibility.</p> 
+				 <ul> 
+					 <li>
+				<p>Long sentences:</p>
+				<aside class="example">Appearance of the text and page layout (color of background and texts, font family and font size as well as spaces between letters, words, sentences or paragraphs) can be modified according to the possibilities offered by the reading system.</aside>
+				<p>or</p>
+				<aside class="example">Text and page layout can not be modified, the reading experience is close to a print version. Reading system can still provide zooming options.</aside>
+				<p>or</p>
+				<aside class="example">Unknown appearence modification possibilities.</aside>
+				</li> 
+				 <li> 
+				<p>Short sentences:</p>
+				<aside class="example">Appearance can be modified.</aside>				
+				<!--if Display transformability is present-->
+				<p>or</p>
+				<aside class="example">Appearance can not be modified.</aside>				
+				<!--if display transformability is not present and text on visual is present-->
+				<p>or</p>
+				<aside class="example">Unknown appearence modification possibilities.</aside>
+				<!--if neither display transformability nor text on visual are presents-->
```

Comment:
correction for spelling of Appearance.
Also, it is better to say Possibilities of modifying Appearance are not known.